### DISCUSSION TITLE
Icon Request: fa-driver (chauffer)

### DISCUSSION THREAD

**Discussion by artsycat** [FortAwesome/Font-Awesome#21190] [Icon Requests]
Hello! I'm a graphic and web designer who's developing a WordPress website for a highway safety organization. I'm customizing the Gateway theme which makes it really easy to use your icons, thanks to a shortcode that you can directly insert in the visual and text editor. Unfortunately, the organization has already a set of icons for each different category of its conference workshops, but they are not responsive, as you can see on this test page hosted on my design partner's web account:
http://mamulligan.com/WPtest/workshops/

I was wondering if you could kindly recreate some of these icons in your own style so we can use them with the theme. I was able to find a desktop computer, a drinking glass, a road and a graduation cap icons among your existing ones, so I would need a driver with a seat belt, a police badge, an eye with a forbidden sign in it, a child in a car seat, and a pedestrian. 

Neither I nor my design partner have an extensive web development background, so our coding skills are limited and we can't figure out how to use the HTML and JavaScript code that an online responsive icon generator provided us with when we tried to create our own SVG icons. We have no idea how to use the code in WordPress, or even if that's possible at all. Therefore, we would be truly grateful if you could add those icons to your repertoire so we can use them. We're sure they would come in handy for many other users, too, as they're pretty standard! 

Thanks so much!

**Comment by devgar** [comment]
+1

**Comment by el-duderino5** [comment]
+1

**Comment by ivantrouvetout** [comment]
![image](https://user-images.githubusercontent.com/10398391/46400398-a3fa1780-c6fa-11e8-8748-1830f7395662.png)

**Comment by cemilf** [comment]
May be driver icon can be a derived version of [user-tie](https://fontawesome.com/icons/user-tie?style=solid) icon with a driver hat.

![driver](https://user-images.githubusercontent.com/1927295/68068778-fed0c900-fd69-11e9-95c1-29dd8e8134af.png)

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
